kview/src/main/resources/dev/ikm/komet/kview/mvvm/view/navigation/pattern-nav-entry.fxml

```diff
@@ -35,29 +32,7 @@
                         <SVGPath content="M0 2a2 2 0 1 0 4 0a2 2 0 1 0 -4 0 M7 2a2 2 0 1 0 4 0a2 2 0 1 0 -4 0 M7 9a2 2 0 1 0 4 0a2 2 0 1 0 -4 0 M0 9a2 2 0 1 0 4 0a2 2 0 1 0 -4 0 M0 16a2 2 0 1 0 4 0a2 2 0 1 0 -4 0 M7 16a2 2 0 1 0 4 0a2 2 0 1 0 -4 0" fill="#9a9da0" stroke="#9a9da0" strokeLineCap="ROUND" strokeLineJoin="ROUND" />
                      </graphic>
                      <contextMenu>
-                        <ContextMenu fx:id="contextMenu" styleClass="search-context-menu-container">
-                           <items>
-                              <MenuItem mnemonicParsing="false" styleClass="search-context-menu-item" text="Add New Instance">
-                                 <graphic>
-                                    <Region styleClass="add-new-instance-icon">
-                                       <padding>
-                                          <Insets right="16.0" />
-                                       </padding>
-                                    </Region>
-                                 </graphic>
-                              </MenuItem>
-                              <SeparatorMenuItem mnemonicParsing="false" styleClass="search-option-menu-line" />
-                              <MenuItem mnemonicParsing="false" styleClass="search-context-menu-item" text="Remove">
-                                 <graphic>
-                                    <Region styleClass="remove-icon">
-                                       <padding>
-                                          <Insets right="16.0" />
-                                       </padding>
-                                    </Region>
-                                 </graphic>
-                              </MenuItem>
-                           </items>
-                        </ContextMenu>
+                        <ContextMenu fx:id="contextMenu" styleClass="search-context-menu-container" />
```

Comment:
Can we remove line 35 and the surrounding `</contextMenu>`?

Reply:
@bhharsh13 Can we also have @dukke review to make sure I'm not missing something?

Thank you! 👍🏼

Reply:
@bhharsh13  I agree with what @dukke is suggesting.

In other words, move the context menu (and options) for the **Pattern row** and not the six dots. I believe this ticket stating to remove options. We want the context menu to work just not on the six dots icon.

We should update the ticket story and acceptance criteria to reflect this. If there is already a ticket It should be combined. 
**A possible user story title:**
As a user, i can perform a context menu popup for the Pattern (row) as a whole and not on the six buttons.